src/plugins/folderView/components/ListView.vue

```diff
@@ -121,6 +133,17 @@ export default {
                 this.sortBy = field;
                 this.ascending = defaultDirection;
             }
+
+            window.localStorage
+                .setItem(
+                    'list-sort-order',
```

Comment:
let's prefix this with something to provide a namespace (and avoid a potential naming collision). openmct-list-sort-order perhaps?

Reply:
I named it openmct-listview-sort-order